doc/telemetry.rst

```diff
@@ -97,81 +99,144 @@ Go Runtime metrics
-dashes and <OpName> the TOSCA operation name where dots where replaced by dashes.
-
-+--------------------------------------------------------------------+--------------------------------------------------+---------------------+-------------+
-|                            Metric Name                             |                   Description                    |        Unit         | Metric Type |
-|                                                                    |                                                  |                     |             |
-+====================================================================+==================================================+=====================+=============+
-| ``yorc.executor.<ExecType>.<DepID>.<NodeType>.<OpName>``           | This measures the duration of an execution.      | milliseconds        | timer       |
-+--------------------------------------------------------------------+--------------------------------------------------+---------------------+-------------+
-| ``yorc.executor.<ExecType>.<DepID>.<NodeType>.<OpName>.failures``  | This counts the number of failed executions.     | number of failures  | counter     |
-+--------------------------------------------------------------------+--------------------------------------------------+---------------------+-------------+
-| ``yorc.executor.<ExecType>.<DepID>.<NodeType>.<OpName>.successes`` | This counts the number of successful executions. | number of successes | counter     |
-+--------------------------------------------------------------------+--------------------------------------------------+---------------------+-------------+
+~~~~~~~~~~~~~~~~~~~~~~
+
+There are two types of executors in Yorc: ``delegate`` executors and ``operation`` executors. 
+Delegate executors handle the deployment of Yorc natively supported TOSCA nodes (like an OpenStack compute for instance).
+Operation executors handle implementations of an lifecycle operation provided as part of the TOSCA node definition (like a shell script or an ansible playbook).
+
+In the below table <ExecType> is the executor type (``operation`` or ``delegate``).
+
+
++---------------------------------------+-------------------+------------------------------------------------+---------------------+-------------+
+|           Metric Name                 |     Labels        |                Description                     |      Unit           | Metric Type |
+|                                       |                   |                                                |                     |             |
++=======================================+===================+================================================+=====================+=============+
+| ``yorc.executor.<ExecType>``          | Deployment        | This measures the duration of an execution.    | milliseconds        | timer       |
```

Comment:
Here I would rename `yorc.executor.<ExecType>` to be more precise about what this metric provides, something like `yorc.executor.<ExecType>.duration`